### DISCUSSION TITLE
Code that checks for current time using FlxG.sound.music.time doesnt work

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2626] [Q&A]
![image](https://user-images.githubusercontent.com/60806448/185383479-028fe887-2003-4488-b8e3-9f6c300e285c.png)

if 2 values are equal, a VISIBLE object should spawn, but it does not

![image](https://user-images.githubusercontent.com/60806448/185383631-0a21ad33-0a83-439b-897a-b3bad52fd3c1.png)

here's how first few values look

**Comment by Geokureli** [comment]
This seems like an issue highly specific to FNF and it's hard to tell whether this is a Flixel bug or a FNF bug. Either reduce the example to a small snippet that only depends on flixel utilities, or ask in a FNF forum
